.storybook/isChromatic.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function isChromatic(): boolean;
```

Comment:
would it not have been better to convert our `isChromatic.js` to a `.ts` equivalent rather than making a separate `.d.ts` file?

Reply:
Our current typescript setup would not allow this unfortunately since we are only compiling `.ts` files only within `src/`. I believe we tried to amend this before but encountered further issues, so I think we should have a dedicated investigation task for this

Reply:
ah OK fair enough - if it's been tried and doesn't work, then happy for an investigation task to be raised and to use the .d.ts as a workaround.

Reply:
Raised task as FE-5398